blog/_posts/2022-01-17-scala-3.1.1-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! After a slightly longer than usual pause, we are back with a new release of the Scala compiler and exciting news about improvements in forward compatibility. The delay in the release was caused by urgent bugs that needed backporting and testing, and then the holiday season.
+
+## Improvements is Scala 3.1.1
+
+Scala 3.1.1-RC2 was promoted to a full stable release of the language. Scala 3.1.1 is the first patch version after the 3.1 minor release. It is therefore focused on bug fixes and improvements in overall correctness and stability. Although there are no new additions to the language, you can still spot some noticeable improvements:
+
+- You can use `using` clauses to introduce Scala 2 conversions.
+
+  Now the code like this should compile and perform necessary conversion from `A` to `B` in the method body:
+
+  ```scala
+    def example[A, B](a: A)(using A <:< B) =
+      val b: B = a
+      ???
+  ```
+
+  To know more see [PR #13662](https://github.com/lampepfl/dotty/pull/13662) and [related issue](https://github.com/lampepfl/dotty/issues/12955).
+- Mirrors can be correctly summoned for hierarchical sum types compiled by Scala 3.0.x ([PR #14100](https://github.com/lampepfl/dotty/pull/14100)).
```

Comment:
If this is important enough to release note, I think the wording could do more to convey what the actual importance of this is.

Reply:
@bishabosha Does that make sense now?